pref/pref.js

```diff
@@ -86,7 +86,7 @@ function savePrefs()
         }
         if (window.addonConfig[key].type == 'bool')
         {
-            preferences[key] = inputElem.checked || false;
+            preferences[key] = inputElem.checked;
```

Comment:
Was there a reason for ORing with False here? Maybe I shouldn't have bothered changing this